//= error code 2

fun main(): Unit {
    std::exit(2i32);
}